data/json/recipes/recipe_deconstruction.json

```diff
@@ -2276,7 +2276,7 @@
       [ [ "cable", 12 ] ],
       [ [ "chem_rdx", 20 ] ],
       [ [ "circuit", 6 ] ],
-      [ [ "plut_cell", 6 ] ],
+      [ [ "raw_plut", 6 ] ],
       [ [ "power_supply", 1 ] ],
       [ [ "scrap", 200 ] ],
       [ [ "small_storage_battery", 2 ] ]
```

Comment:
I think it would be nice if disassembling mininukes gave something rare of gameplay benefit, since they are a lab finale item.  Maybe replace the `small_storage_battery` with `plut_cell` now that those are no longer standing in for the fissile material.  I think it makes sense that such items would use `plut_cell` as their power source.